lstchain/mc/plot_utils.py

```diff
@@ -0,0 +1,162 @@
+import matplotlib.pyplot as plt
+from lstchain.spectra.crab import crab_HEGRA
+
+def fill_bin_content(ax, energy_bin, gb, tb):
+    """
+
+    Parameters
+    --------
+
+    Returns
+    --------
+    """
+    for i in range(0,gb):
+        for j in range(0,tb):
+            text = ax.text((j+0.5)*0.05, (i+0.5)*0.1, "%.2f %%" % sens[energy_bin][i][j],
+                       ha="center", va="center", color="w")
+    return ax
+
```

Comment:
Uses "sens" but is not given as function argument

Reply:
Done